src/emulator/auth/server.ts

```diff
@@ -373,9 +373,8 @@ export async function createApp(
       const errorString =
         `Multiple projectIds are not recommended in single project mode. ` +
         `Requested project ID ${projectId}, but the emulator is configured for ` +
-        `${defaultProjectId}. This warning will become an error in the future. To opt-out of ` +
-        `single project mode add/set the \'"singleProjectMode"\' false' property in the` +
-        ` firebase.json emulators config.`;
+        `${defaultProjectId}. To opt-out of single project mode add/set the ` +
+        `\'"singleProjectMode"\' false' property in the firebase.json emulators config.`;
```

Comment:
I don't see that this message is actually changing the flag name. Is that correct?

Reply:
you're right - that's odd. The repo flag is single_project_mode though I checked.